test/src/test/java/org/corfudb/infrastructure/log/StreamLogFilesTest.java

```diff
@@ -370,6 +373,7 @@ public void testStreamLogDataCorruption() throws Exception {
 
         final int OVERWRITE_BYTES = 4;
 
+        String logDir = getContext().getConfiguration().getLogDir();
```

Comment:
Since `getContext()` allocates a new `ServerContext`, can it be reused instead of creating multiple copies in the same test?

Reply:
I think that it is present to keep separate tests in the class independent from each other, in case some future test added needs to modify the configuration.

Reply:
I agree that each test should call this method for the reason you mention, but some tests call `getContext` multiple times.

Reply:
Oh got it, you're suggesting creating a local variable in the test method.